insightfinder/manifest.json

```
@@ -0,0 +1,34 @@
+{
+  "display_name": "InsightFinder",
+  "maintainer": "support@insightfinder.com",
+  "manifest_version": "1.0.0",
+  "name": "insightfinder",
+  "metric_prefix": "insightfinder.",
+  "metric_to_check": "",
+  "creates_events": false,
+  "short_description": "Integrate data from DataDog for analysis by InsightFinder",
+  "guid": "1ecaeadf-7b34-48f7-b9cb-81b8a9e4e4dc",
+  "support": "contrib",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "InsightFinder",
+  "categories": [
+    "automation",
+    "monitoring",
+    "notification"
+  ],
+  "type": "check",
+  "is_public": false,
```

```suggestion
  "is_public": true,
```
Would you like the integration to be publicly available on our documentation after merging this PR? If so, you can do so by changing this field.